Products/CMFPlone/tests/test_okay.py

```diff
@@ -0,0 +1,53 @@
+# -*- coding: utf-8 -*-
+from plone.testing.z2 import Browser
+from Products.CMFPlone.testing import PRODUCTS_CMFPLONE_INTEGRATION_TESTING
+
+import unittest
+
+
+class OkayTest(unittest.TestCase):
+    """Test the OK simple status view.
```

Comment:
I *did* look at that missing blank line **after**, thinking it might be nicer to add one. Done.